style.css

```diff
@@ -0,0 +1,147 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+main {
+  background: #e5e5e5;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  position: fixed;
+  background-color: white;
+  z-index: 1000;
+}
+
+nav img {
+  margin: 10px;
+  width: 45px;
+}
+
+#sec1 {
+  position: relative;
+  background: white;
+  height: 718px;
+  border-bottom-left-radius: 20%;
+  padding-top: 180px;
+}
+
+.shape1 .rect {
+  position: absolute;
+  left: 0;
+  bottom: 5px;
+}
+
+.shape2 {
+  position: relative;
+  left: 60%;
+  width: 0;
+}
+
+.shape2 .rect {
+  position: relative;
+  left: -14px;
+  bottom: 16px;
+}
+
+.shape3 {
+  position: absolute;
+  right: 0;
+}
+
+.shape3 .rect {
+  position: absolute;
+  bottom: 70px;
+  right: 1px;
+}
+
+.shape4 {
+  position: relative;
+  width: 0;
+  height: 0;
+  left: 20%;
+  top: 10%;
+}
+
+.shape5 {
+  position: absolute;
+  width: 0;
+  height: 0;
+  right: 25%;
+}
+
+.shape5 .rect {
+  position: absolute;
+  bottom: -20px;
+}
+
+.shape6 {
+  position: absolute;
+  right: 300px;
+  bottom: 300px;
+}
+
+.shape6 .ecl {
+  position: relative;
+  top: 13px;
+  left: 19px;
+}
+
+#sec1 article {
+  width: 90%;
+  margin: auto;
+}
+
+#logo {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+  margin: 10px;
+  padding: 8px;
+}
+
+/* Section */
+#nm {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #344563;
+  margin-bottom: 10px;
+}
+
+.desc {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  margin-top: 15px;
```

Comment:
Hi @GedeonTS, In this problem i am using the same font charachtestics, but i can't find any specific paddings or margins in the figma design, the only difference is that figma uses flex to center the text and i use margin auto.

Reply:
Hello @codedit334, right now when you tried to fix that, the design got bugs and now it is far from similar to the Figma design. I would suggest you to use details of font sizes, margins, and paddings from Figma design for every element in your project so that it would be pixel-perfect.